libraries/common/subscriptions/router.js

```diff
@@ -41,6 +42,13 @@ export default function router(subscribe) {
       location = location.slice(0, -1);
     }
 
+    // Check for a redirect url and change location if one is found.
+    const redirect = redirects.get(location);
```

Comment:
There is an issue when protected routes are redirected. The system doesn't work anymore - see theme pr.
Additionally you should update the unit test for the `router` subscriptions.